css/properties/transition.json

```diff
@@ -0,0 +1,239 @@
+{
+  "css": {
+    "properties": {
+      "transition": {
```

Comment:
The comments about transition-delay.json are also valid here. Please apply them here too.